pallets/liquid-staking/src/lib.rs

```diff
@@ -1652,6 +1700,30 @@ pub mod pallet {
             Ok(())
         }
 
+        #[require_transactional]
+        fn do_loans_fast_unstake(who: &AccountIdOf<T>, amount: BalanceOf<T>) -> DispatchResult {
+            let loans_fast_unstake_fee = T::LoansFastUnstakeFee::get()
+                .checked_mul_int(amount)
+                .ok_or(ArithmeticError::Overflow)?;
```

Comment:
Is there any connection between `LoansFastUnstakeFee` and `CollateralCurrency.borrow_rate`?
because this affects a lot, whether LiquidStaking pallet can repay_all account_borrows.

Reply:
@MrPai actually we repay only partially not the whole account_borrows. I'm think maybe we should use `max` instead of `min` to not left KSM/DOT inside liquid staking